internal/cmd/gocdk/static/demo/blob.bucket/demo_blob_bucket.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2019 The Go Cloud Authors
```

Comment:
Won't this add a copyright header to code generated for the user? I don't believe we want that.

Also, on the location of this file:

1. It's not under an inner internal directory, so when we eventually move internal/cmd/ -> cmd/, then anyone could import this package.
2. This will get built when `./...` is matched, which is not what we want. The easiest hack is to add an underscore to the beginning of the static directory (see very last sentence of [Package lists and patterns](https://golang.org/cmd/go/#hdr-Package_lists_and_patterns) for guarantee that this works).

Reply:
Removed copyright.

Renamed directory to `_static`.